vcluster/configure/vcluster-yaml/sync/from-host/custom-resources.mdx

```diff
@@ -34,6 +31,151 @@ sync:
         scope: Cluster
 ```
 
+## Namespaced scope custom resource definitions
+By default, this is turned off.
+
+Enabling this allow you to sync namespaced CustomResources from the specified namespaces in the host to the specified namespaces in the vCluster.
+
+It is also possible to modify the name of the synced resource in the virtual cluster.
+
+There is no option to sync from all namespaces in the host.
+
+Sync is one-directional, from host to virtual. If you modify an object in the host, vCluster syncs the change to virtual object. When you delete virtual object, vCluster re-creates it if the host object still exist.
+
+When you delete host object, vCluster deletes virtual object.
+
+### Prerequisites
+
+* All the specified namespaces have to exist in the host at the vCluster startup.
+* Custom Resource Definition needs to exist in the host at the vCluster startup.
+
+### Example
+
+:::info No need to configure RBAC
+vCluster automatically adds the required cluster RBAC permissions for retrieving the custom resource definition and syncing the resources from the host to the virtual cluster.
+:::
+
+For Namespaced Scoped Custom Resources, you need to specify `selector.mappings` in the config. This tells vCluster which host resources should be synced and where (in the virtual cluster).
+
+Namespaces in the virtual cluster are created automatically during the sync (if they do not exist already).
+
+It is not possible to sync Custom Resources that were already synced from virtual to host, they are skipped by vCluster.
+
+Here is an example with cert-manager’s CertificateRequest custom resource.
+
+To sync all CustomResources from a given namespace in the host to the given namespace in the virtual cluster, “namespace/*” wildcard can be used, e.g.:
+
+```yaml
+sync:
+  fromHost:
+    customResources:
+      certificaterequests.cert-manager.io:
+        enabled: true
+        scope: Namespaced
+        selector:
+          mappings:
+	          # syncs all CertificateRequests from "foo" namespace
+	          # to the "bar" namespace in vCluster. CertificateRequests names are unchanged.
+            "foo/*": "bar/*"
```

Comment:
The two commented lines are wrong indentation

Reply:
indentation is ok, it's github displays this weirdly